### ISSUE TITLE
Overlapping ME Storage bus filters break on the same network

### ISSUE THREAD

**Issue by Niphemy** [AE2-UEL/Applied-Energistics-2#489]
Multiple storage buses with overlapping filters result in incorrect numbers of items when reading from another network

**To Reproduce**
*Note the specific items used are irrelevant
Create three storage buses with different but overlapping filters:
![image](https://github.com/user-attachments/assets/ba4c22e8-b38b-473e-a57c-74f69fd2fbd7)
![image](https://github.com/user-attachments/assets/ada25707-559d-4d0a-8f52-b6d53f0b184e)
![image](https://github.com/user-attachments/assets/541dcdeb-6b38-417f-b08e-d0a9a44c2438)

Connect these storage buses to another network via ME interface. This network stores the filtered items
![image](https://github.com/user-attachments/assets/3a75b850-bc5f-4783-ac81-8654fcb7c06d)

Upon observing the items reported by the original network the counts are not reported correctly
![image](https://github.com/user-attachments/assets/d3746f77-f30a-467b-be14-dc495d8164c8)

Additionally, Depending on how you connect the cables different totals are reported
![2024-09-07_22 49 33](https://github.com/user-attachments/assets/181d6bab-b004-4ebf-b28b-ffb6f56d0a77)

**Expected behavior**
The Storage bus should always report 4M for each filtered item that it can see through the interface.

**Additional context**
A similar issue is reported on the original Applied Energistics Repo that looks like it was fixed July 2024
https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/8018

**Environment**
Singleplayer
NomiFactory 1.7-alpha-4

- Minecraft Version: 1.12.2
- AE2 Version: AE2-UEL v56.6.0
- Forge Version: 8.0.99.99

**Comment by NotMyWing** [comment]
I agree that it should at least be consistent, but why would over-reporting be desired?

**Comment by Niphemy** [comment]
To clarify the desired behaviour is that the system reports the quantities listed on the chests
